cmd/skupper/skupper.go

```diff
@@ -210,11 +209,17 @@ installation that can then be connected to other skupper installations`,
 			}
 
 			if routerModeFlag.Changed {
-				options := []string{routerModeInteriorValue, routerModeEdgeValue}
+				options := []string{string(types.TransportModeInterior), string(types.TransportModeEdge)}
```

Comment:
TransportModeEdge seems to be missing the TransportMode type at `types.go`

Reply:
hmm, I see that it is present